### PR TITLE
add CI workflow for GitHub Actions

### PR DESCRIPTION
**Pull request by tdyas**
Add a CI workflow to test the code in this repository using GitHub Actions.

### CODE REVIEW

File: .github/workflows/pants.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Bootstrap Pants
+      run: |
+        ./pants --version
+    - name: Check BUILD files
+      run: ./pants tailor --check update-build-files --check
+    - name: Lint and typecheck
+      run: | 
+        ./pants lint typecheck ::
```
Comment (Eric-Arellano) [inline .github/workflows/pants.yaml:52]: We don't activate any linters and typecheckers, so imo this should be removed

File: .github/workflows/pants.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - name: Test
+      run: |
+        ./pants test ::
+    - name: Package / Run
+      run: |
+        ./pants package ::
```
Comment (Eric-Arellano) [inline .github/workflows/pants.yaml:58]: Ditto, nothing to package

**Review by Eric-Arellano — APPROVED** [review]

File: .github/workflows/pants.yaml (hunk 1 above)
Comment (tdyas) [reply .github/workflows/pants.yaml]: Removed.

**Review by Eric-Arellano — APPROVED** [review]

File: .github/workflows/pants.yaml (hunk 2 above)
Comment (tdyas) [reply .github/workflows/pants.yaml]: Removed.

**Pull request merged**